src/components/tables/EntityTable.tsx

```diff
@@ -185,6 +181,12 @@ function EntityTable({
             setPagination(getPagination(newPage, rowsPerPage));
             setPage(newPage);
         },
+        changeRowsPerPage: (event: ChangeEvent<HTMLInputElement>) => {
+            const newLimit = parseInt(event.target.value, 10);
+            setRowsPerPage(newLimit);
+            setPagination(getPagination(0, newLimit));
+            setPage(0);
```

Comment:
Was hoping to use the handler above to set the pagination stuff but that did not work. So just had to duplicate it.